tracing-core/src/callsite.rs

```diff
@@ -114,7 +114,7 @@ pub struct Identifier(
 /// [`enabled`] is evaluated for every event.
 ///
 /// This function will also re-compute the global maximum level as determined by
-/// the [`Subscriber::max_level_hint`] method. If a [`Subscriber`]
+/// the [`Subscriber::max_level_hint`](crate::Subscriber::max_level_hint) method. If a [`Subscriber`]
```

Comment:
elsewhere, we tend to put link references at the end of the doc comment, rather than inline. i have a slight preference for doing this consistently with the newly-added link references, but i'm not going to block merging this on that. 

also, we've tried to wrap the doc comments at a consistent column, can we continue doing that with the changes here?

Reply:
> we tend to put link references at the end of the doc comment, rather than inline
I've fixed this!

Reply:
> we've tried to wrap the doc comments at a consistent column, can we continue doing that with the changes here?

What exactly do I need to do?